tests/Fixer/Basic/OctalNotationFixerTest.php

```diff
@@ -55,7 +55,7 @@ public static function provideFixCases(): iterable
         ];
 
         yield [
-            '<?php $foo = 0;',
+            '<?php $foo = 0o0000;',
```

Comment:
leading zeros in `0[box]` and float numbers should be fixed by a separate fixer

Reply:
yet, if they are fixed already by existing fixer, we should not drop that part of fixing if there is no replacement available.

Reply:
It is minor, see https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7081#discussion_r1250782039, non-octal notation is not fixed